### PR TITLE
Introduce Scarlet / Violet new items

### PR DESCRIPTION
**Pull request by giginet**
I added a new item since Scarlet / Violet.

All data are migrated from here.
[PMSV/Item Names.txt at main · Ruimusume/PMSV](https://github.com/Ruimusume/PMSV/blob/main/Item%20Names.txt)

I corrected some Japanese and English names.

### CODE REVIEW

File: data/v2/csv/items.csv [hunk 1]
```diff
@@ -1606,5 +1606,448 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
 1656,carrot-seeds,22,20,,
 1657,ability-patch,26,20,,
 1658,reins-of-unity,22,0,,
+1659,adamant-crystal,,,,
```
Comment (giginet) [inline data/v2/csv/items.csv]: What number of ID should I start from?

Some items are already available since Legends Arceus.

Comment (Naramsim) [reply data/v2/csv/items.csv]: I think the numbering should be consequent, so it's fine the one used is fine.

Comment (giginet) [reply data/v2/csv/items.csv]: Do you mean, the current numbering doesn't have any problem?

Comment (Naramsim) [reply data/v2/csv/items.csv]: Hi, yes your numbering is good. Anyways, I'm not sure if we can leave empty the category_id, cost,fling_power and fling_effect_id. Don't you have at least the category_id for your new data.

Comment (giginet) [reply data/v2/csv/items.csv]: There are no original data for category_id so I have to fill this column by hand.

Can we add new `item_categories` like `tm_materials` or `sandwich_ingredients`?

Comment (Naramsim) [reply data/v2/csv/items.csv]: we can add new categories if needed, yes

**Comment by giginet** [comment]
I'll add category_ids. However, this column is not necessary for my use case.
  So I'll update this PR, but it will be little late.

**Comment by giginet** [comment]
@Naramsim 

Hi. I added new `item_categories` and filled all `item_category` columns.
Could you review?

File: data/v2/csv/items.csv [hunk 2]
```diff
@@ -1606,5 +1606,448 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
...
+1668,miraidon’s-poké-ball,20,0,,
+1669,tera-orb,20,0,,
+1670,scarlet-book,20,0,,
+1671,violet-book,20,0,,
+1672,kofu’s-wallet,22,0,,
+1673,tiny-bamboo-shoot,24,375,,
+1674,big-bamboo-shoot,24,1500,,
+1675,scroll-of-darkness,21,0,,
+1676,scroll-of-waters,21,0,,
+1677,,,,,
```
Comment (Naramsim) [inline data/v2/csv/items.csv]: Is it normal that is all empty?

**Comment by Naramsim** [comment]
Thanks for the PR, before merging I'd like to have a buildable `master`, so I'd prefer to wait for this: https://github.com/PokeAPI/pokeapi/pull/807#issuecomment-1371509272

File: data/v2/csv/items.csv (hunk 2 above)
Comment (giginet) [reply data/v2/csv/items.csv]: I migrated from the original data. It seems to be missing numbers. These columns should be removed.

I'll remove these columns and fill the following IDs.

Comment (giginet) [reply data/v2/csv/items.csv]: @Naramsim I removed missing columns and filled the following IDs.

https://github.com/PokeAPI/pokeapi/pull/787/commits/ba8ff8fa3d4ab7f7d4250c535ee8f41745f5310f

**Comment by Naramsim** [comment]
Hi @giginet , would it be difficult for you to provide us with the `cost` of the items?

**Comment by giginet** [comment]
I don't have any canonical data about costs.
So I have to fill them by hand from Wiki pages.

The cost column is necessary for merging this?
I'd like to merge this first and add costs after merging.

**Comment by Naramsim** [comment]
For us, it would be better to have the cost since right now it's a mandatory field. I can turn it to a `null`able field though.

**Comment by giginet** [comment]
OK. I'll try to fill in all the cost columns in my spare time.

Could you review other PRs first?

**Comment by Naramsim** [comment]
I'll check out some other PRs :)

**Comment by giginet** [comment]
@Naramsim Hi. I filled all cost columns as far as I know.
https://github.com/PokeAPI/pokeapi/pull/787/commits/bb56e09af5287434c2ac74ef094b4371067902bb

Could you review this?

**Comment by Naramsim** [comment]
Hi @giginet , there's a problem. The `id`s in `items.csv` and `item_names.csv` differ. 

For example `white-dish` is `2098` and `2100`

**Comment by giginet** [comment]
@Naramsim Sorry for not checking enough.
I fixed on this PR.
https://github.com/PokeAPI/pokeapi/pull/819

**Comment by Naramsim** [comment]
> @Naramsim Sorry for not checking enough.
> I fixed on this PR.
> https://github.com/PokeAPI/pokeapi/pull/819

All good! I tested everything locally and it builds fine! 

All your PR are now merged! Thanks so much for the efforts! I'll add your name on our website as a contributor! 

Now we can update the `id`s

**Pull request merged**